src/libraries/Common/tests/Tests/System/StringTests.cs

```diff
@@ -7629,6 +7629,17 @@ public static void InternalTestAotSubset()
 #pragma warning restore 0618 // restore warning when accessing obsolete members
         }
 
+        [Fact]
+        public static void InternTestCanReturnNull()
+        {
+            for (int i = 0; i < 20; i++)
```

Comment:
2 would be sufficient to validate what we're trying to validate, right? The 20 here is just for good measure? Just checking :)

Reply:
(Or, I guess you're just testing 20 different strings because you're calling NewGuid().ToString() each time. Nevermind.)